### PR TITLE
[staging-next] pythonPackages.datadog: fix tests

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 114907` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>2 packages failed to build and are new build failures:</summary>
  <ul>
<li>python37Packages.gradient: log was empty</li>
<li>python38Packages.gradient: log was empty</li>
  </ul>
</details>
<details>
<summary>
6 packages built:
</summary>
<ul>
<li>python37Packages.datadog</li>
<li>python37Packages.gradient_statsd</li>
<li>python38Packages.datadog</li>
<li>python38Packages.gradient_statsd</li>
<li>python39Packages.datadog</li>
<li>python39Packages.gradient_statsd</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
python37Packages.datadog:
<details>


warning: maintainers-missing
Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/datadog/default.nix:56:3:

```
   |
56 |   meta = with lib; {
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
python37Packages.gradient:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python37Packages.gradient_statsd:
<details>


warning: unused-argument
Unused argument: `boto3`.
Near pkgs/development/python-modules/gradient_statsd/default.nix:2:3:

```
  |
2 | , boto3, requests, datadog, configparser, python
  |   ^
```
</details>
python38Packages.datadog:
<details>


warning: maintainers-missing
Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/datadog/default.nix:56:3:

```
   |
56 |   meta = with lib; {
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
python38Packages.gradient:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python38Packages.gradient_statsd:
<details>


warning: unused-argument
Unused argument: `boto3`.
Near pkgs/development/python-modules/gradient_statsd/default.nix:2:3:

```
  |
2 | , boto3, requests, datadog, configparser, python
  |   ^
```
</details>
python39Packages.datadog:
<details>


warning: maintainers-missing
Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/datadog/default.nix:56:3:

```
   |
56 |   meta = with lib; {
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
python39Packages.gradient_statsd:
<details>


warning: unused-argument
Unused argument: `boto3`.
Near pkgs/development/python-modules/gradient_statsd/default.nix:2:3:

```
  |
2 | , boto3, requests, datadog, configparser, python
  |   ^
```
</details>
</details>

**Review by SuperSandro2000 — APPROVED** [review]
not sure about the target branch but otherwise LGTM

**Comment by dotlambda** [comment]
> not sure about the target branch

It's not broken on master, that's why this targets staging-next.

**Comment by dotlambda** [comment]
> This broke `nixops`

It didn't. `python2.pkgs.datadog` was broken before this change.
